BParkingNano/python/electronsBPark_cff.py

```diff
@@ -44,15 +44,15 @@
   mvaId = cms.InputTag("lowPtGsfElectronLatestID"),
   vertexCollection = cms.InputTag("offlineSlimmedPrimaryVertices"),
   ## cleaning wrt trigger muon [-1 == no cut]
-  drForCleaning_wrtTrgMuon = cms.double(-1.),
-  dzForCleaning_wrtTrgMuon = cms.double(-1.),
+  drForCleaning_wrtTrgMuon = cms.double(0.03),
+  dzForCleaning_wrtTrgMuon = cms.double(1.),
   ## cleaning between pfEle and lowPtGsf
-  drForCleaning = cms.double(0.01),
-  dzForCleaning = cms.double(0.01),
+  drForCleaning = cms.double(0.03),
+  dzForCleaning = cms.double(0.7), ##keep tighter dZ to check overlap of pfEle with lowPt (?)
   ## true = flag and clean; false = only flag
   flagAndclean = cms.bool(False),
-  pf_ptMin = cms.double(1.),  ## move to 2 next
-  ptMin = cms.double(1.),
+  pf_ptMin = cms.double(1.),
```

Comment:
bump to 2

Reply:
why dzForcleaning_wrtTrgMuon has two values? probably a git stuff but better check. Also do we have plots for these numbers?

Reply:
I see only one dzForcleaning_wrtTrgMuon (the other is dr -> drForCleaning_wrtTrgMuon)